.changelog/16015.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+connect: add flags `envoy-ready-bind-port` and `envoy-ready-bind-address` to the `consul connect envoy` command that allows configuration of readiness probe on proxy for any service kind.
+```
```

Comment:
I realize now that we have this changelog twice now. We can delete the one on the ENT side, `.changelog/4103.txt`. 
 since this is an OSS feature. My bad for not thinking of that before.

Reply:
on it!

Reply:
https://github.com/hashicorp/consul-enterprise/pull/4131